src/pipeline.ml

```diff
@@ -26,6 +26,18 @@ let master_distro = Distro.((resolve_alias master_distro : distro :> t))
 
 type 'a run = ('a, unit, string, Dockerfile.t) format4 -> 'a
 
+let maybe_add_overlay (run : 'a run) distro switch =
```

Comment:
Since this is temporary workaround, a comment with a TODO pointing to the tracking issue for the proper would help with traceability here.